mercury/views/measurement.py

```diff
@@ -12,12 +12,42 @@ def build_error(str):
     return json.dumps({"error": str})
 
 
+def add_measurement(request, event):
+    json_data = request.data
+    if isinstance(json_data, str):
+        json_data = json.loads(json_data)
+
+    res = {"event_uuid": event.uuid}
+    dic = {
+        "timestamp": "date",
+        "sensor_id": "sensor_id",
+        "value": "values",
+    }
+
+    for d in dic:
+        if json_data.get(dic[d]) is None:
```

Comment:
Optional: name suggestion for readability

```
key_map = {                           
    "timestamp": "date",              
    "sensor_id": "sensor_id",         
    "value": "values",                
}                                     
                                      
for key, json_key in key_map.items(): 
    if json_key not in json_data:     
        return Response(...)          
    res[key] = json_data[json_key]    
```

Reply:
Nice suggestion! I have updated the code.